Day-5/main.js

```diff
@@ -0,0 +1,112 @@
+// Q: What is the difference between ++i and i++?
+// ANSWER:
+let i = 0;
+console.log(++i); // OUTPUT: 2
```

Comment:
this should be printing 0

Reply:
My Mistake,  will change.

Reply:
`++i will increment the value of i by 1 and return 2.` Change this to `++i will increment the value of i by 1 and return 1.`
As when we use ++i, it will increment the value to 1 which happen before returning the value. So console log will print   1 and if we try to read the value of `i` it will be also 1.